packages/buidler-core/src/internal/cli/sentry/transport.ts

```diff
@@ -0,0 +1,37 @@
+import { Event, Response } from "@sentry/node";
+import { fork } from "child_process";
+
+// This class is wrapped in a function to avoid having to
+// import @sentry/node just for the BaseTransport base class
+export function getSubprocessTransport(
+  verbose: boolean,
+  configPath: string | undefined
+): any {
+  const { Status, Transports } = require("@sentry/node");
+
+  class SubprocessTransport extends Transports.BaseTransport {
+    public async sendEvent(event: Event): Promise<Response> {
+      const serializedEvent = JSON.stringify(event);
+
+      const env: Record<string, string> = {
+        BUIDLER_SENTRY_EVENT: serializedEvent,
+        BUIDLER_SENTRY_VERBOSE: verbose.toString(),
+      };
+
+      if (configPath !== undefined) {
+        env.BUIDLER_SENTRY_CONFIG_PATH = configPath;
+      }
+
+      fork(`${__dirname}/subprocess`, [], {
```

Comment:
I think this has to be detached, at least in Linux and macos. Here's more info about it: https://nodejs.org/api/child_process.html#child_process_options_detached

I'd also test what `detached` does on windows.

Reply:
Done.